server/protoboards_test.go

```diff
@@ -239,7 +239,7 @@ func Test_ProtoboardsID(t *testing.T) {
 			wants: wants{
 				statusCode:  http.StatusOK,
 				contentType: "application/json",
-				body:        `{"id":"1","meta":{"name":"","version":"","dashboardVersion":""},"data":{"cells":null},"links":{"self":"/chronograf/v1/protoboards/1"}}`,
+				body:        `{"id":"1","meta":{"name":"","version":"","measurements":null,"dashboardVersion":""},"data":{"cells":null,"templates":null},"links":{"self":"/chronograf/v1/protoboards/1"}}`,
```

Comment:
Thanks for the PR! Can you comment on why this test change was needed (feel free to amend this commit for the details).

Reply:
This is required due to changes in the `encoding/json` package between Go 1.21.12 and 1.22.7. I've reviewed the changelogs of Go but was unsuccessful in finding the exact change. From my perspective, this is an acceptable change because our tests already shows that `null` is returned for some empty fields.